.github/workflows/default.yaml

```diff
@@ -0,0 +1,105 @@
+name: Default (push)
+
+on:
+  schedule:
+    - cron: '0 10 * * 0' # Every Sunday at 10AM
+  push:
+    branches:
+      - master
+      - develop
+      - feature/*
+    tags:
+      - 'v*.*.*'
+    paths-ignore:
+      - 'docs/**' # Don't run if there were changes to docs only
+  workflow_dispatch: # Allow manually triggering a build
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  REPO: docksal/docksal
+  DOCKSAL_SSH_AGENT_USE_HOST: 0
+  DOCKSAL_DNS_DISABLED: 1
+  DOCKSAL_DNS_DOMAIN: docksal.site
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false # Don't cancel other jobs if one fails
+      matrix:
+        suite:
+          #- quicktest # Uncomment to run
+          - project
+          - config
+          - db
+          - system
+          - commands
+          #- acquia
+          #- pantheon
+          #- platform
+          #- drush
+          #- wp
+
+    steps:
+      # Hack to fix docker-gen in Github Actions
+      # Drops docker daemon config with custom cgroup (which docker-gen does not support)
+      # See https://github.com/jwilder/docker-gen/issues/315
+      - name: Fix docker-gen in Github Actions
+        run: |
+          sudo rm /etc/docker/daemon.json
+          sudo service docker restart
```

Comment:
With service/vhost-proxy at 1.7 now, this might no longer be needed. Definitely not a release blocker though.